src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,153 @@
 package core.basesyntax;
 
+//import java.util.HashMap;
```

Comment:
Please remove this comment. Comments should not appear in the code except Java docs and some moments when this comment is an explanation for some code.